core/src/components/select/test/fill/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Select - Fill</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+    <style>
+      .grid {
+        display: grid;
+        grid-template-columns: repeat(3, minmax(250px, 1fr));
+        grid-row-gap: 20px;
+        grid-column-gap: 20px;
+      }
+      h2 {
+        font-size: 12px;
+        font-weight: normal;
+
+        color: #6f7378;
+
+        margin-top: 10px;
+      }
+      @media screen and (max-width: 800px) {
+        .grid {
+          grid-template-columns: 1fr;
+          padding: 0;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>Select - Fill</ion-title>
+        </ion-toolbar>
+      </ion-header>
+
+      <ion-content id="content" class="ion-padding">
+        <div class="grid">
+          <div class="grid-item">
+            <h2>No Fill</h2>
+            <ion-select class="no-fill" label="Fruit"></ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Shaped No Fill</h2>
+            <ion-select shape="round" class="no-fill" label="Fruit"></ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Solid</h2>
+            <ion-select fill="solid" label="Fruit" value="apple">
+              <ion-select-option value="apple">Apple</ion-select-option>
+            </ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Shaped Solid</h2>
+            <ion-select shape="round" fill="solid" label="Fruit" value="apple">
+              <ion-select-option value="apple">Apple</ion-select-option>
+            </ion-select>
+          </div>
+
+          <div class="grid-item">
+            <h2>Solid, Floating Label</h2>
+            <ion-select fill="solid" label="Fruit" label-placement="floating" value="apple"></ion-select>
```

Comment:
~ Is there a reason that some of the examples have options and others do not? I know the test does not use this template, but it might be helpful if all the examples render at least 1 option for a consistent display (unless we are wanting to demonstrate a state of no options).

Adding options would also allow manual verification of the visual state after selecting a value.

Reply:
I think that was an oversight on my part. I pushed a commit that ensures all selects have values.